cmd/ci-operator/main.go

```diff
@@ -450,6 +451,7 @@ func (o *options) Complete() error {
 		o.templates = append(o.templates, template)
 	}
 
+	o.clusterConsoleURL = "https://console.svc.ci.openshift.org"
```

Comment:
Huh, hardcoding URLs looks fishy. At minimum, this should be named constant.

Reply:
A new constant defined.